scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1648,6 +1648,11 @@ class Router(formatOps: FormatOps) {
           if isTypeVariant(leftOwner) =>
         Seq(Split(Space(isSymbolicIdent(tok.right)), 0))
 
+      // Kind projector type lambda
+      case tok @ FormatToken(T.Ident("+") | T.Ident("-"), T.Underscore(), _)
+          if leftOwner.is[Type.Name] =>
+        Seq(Split(NoSplit, 0))
+
```

Comment:
```suggestion
      // Kind projector type lambda
      case FormatToken(T.Ident("+" | "-"), _: T.Underscore, _)
          if leftOwner.is[Type.Name] =>
        Seq(Split(NoSplit, 0))

```

Reply:
above, `tok` is unused, and i think `Ident` can be used once.

Reply:
Fixed, good catch!